criu/net.c

```diff
@@ -2437,12 +2426,38 @@ static inline int restore_nftables(int pid)
 	nft_ctx_free(nft);
 buf_free_out:
 	xfree(buf);
+out:
+	return ret;
+}
+#endif
+
+static inline int restore_nftables(int pid)
+{
+	int ret = -1;
```

Comment:
nit: naming convention for variable which we return from current function is `exit_code`, please do same thing in do_restore_nftables too.

Reply:
Have done.